nodejs/week1/src/backend/routes/cheap-meals.js

```diff
@@ -0,0 +1,21 @@
+const mealsRouter = require('./meals.js');
+
+let cheapPrice = 90;
+let cheapMeals = [];

```

Comment:
these two are not being re-assigned so I would recommend using `const` instead :)

https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75